compatibility.md

```diff
@@ -12,29 +12,22 @@ version.
 | Package name | Minimum supported version | Latest supported version | Introduced in* |
 | --- | --- | --- | --- |
 | `@apollo/gateway` | 2.3.0 | 2.8.3 | `@newrelic/apollo-server-plugin@1.0.0` |
-| `@apollo/server` | 4.0.0 | 4.10.4 | `@newrelic/apollo-server-plugin@2.1.0` |
-| `@aws-sdk/client-bedrock-runtime` | 3.0.0 | 3.616.0 | 11.13.0 |
-| `@aws-sdk/client-dynamodb` | 3.0.0 | 3.616.0 | 8.7.1 |
-| `@aws-sdk/client-sns` | 3.0.0 | 3.616.0 | 8.7.1 |
-| `@aws-sdk/client-sqs` | 3.0.0 | 3.616.0 | 8.7.1 |
-| `@aws-sdk/lib-dynamodb` | 3.0.0 | 3.616.0 | 8.7.1 |
-| `@aws-sdk/smithy-client` | 3.0.0 | 3.374.0 | 8.7.1 |
+| `@apollo/server` | 4.0.0 | 4.10.5 | `@newrelic/apollo-server-plugin@2.1.0` |
 | `@elastic/elasticsearch` | 7.16.0 | 8.14.0 | 11.9.0 |
```

Comment:
🤔

Reply:
what's wrong with this one? we have `">=7.16.0`

Reply:
> oh are you referring to all the packages that are removed?

Correct. Some error is popping that really shouldn't be.